Created module for POI. POI intro, examples and Presentation Maker intro (#14)
apluslms/course-templates: m10_point_of_interest/01_introduction.rst

> Point Of Interest (POI)

Reviewer: In order to keep the consistency of the manual, I will suggest starting each chapter with the `styled_topic` directive.

=======================

Introduction to the Point of Interest directive.

> What is it?

Reviewer: ```suggestion
What is a Point of Interest?
```

-----------

Directive for creating a "point of interest" summary block.

> A point of interest is mostly like a normal admonition ("coloured info box"), but

Reviewer: ```suggestion
A point of interest is similar to a normal admonition ("coloured info box"), but
```

> they are also linked to each other with next/previous links. The links enable

Reviewer: ```suggestion
the points of interest are also linked to each other with next/previous links. The links enable
```

> the user to quickly navigate between the points of interest.

Reviewer: ```suggestion
the user to quickly navigate between different points of interest.
```

Point of interests may also be used to generate separate lecture slides
(not directly included in the A+ content chapters). This requires a separate

> tool called "Presentation Maker". More about Presentation Maker in the chapter 10.3.

Reviewer: ```suggestion
tool called :doc:`Presentation Maker </m10_point_of_interest/03_presentation_maker>`
```

.. point-of-interest:: This is a point of interest

>   And here is the content. More examples in the next chapter.

Reviewer: Add the link to the next chapter

From now on I will refer to point of interest as a POI.

Why to use it?
--------------

Highlight important parts and make them easily accessible with links.

Turn your Point of interest into presentation slides with Presentation Maker.

> Gain new options that come with the POI, such as hiding your POI from the A+ Course material and making it only appear in the presentation slides.

Reviewer: This looks like a list. Otherwise, connect your ideas and write a paragraph.
```suggestion
#. Highlight important parts and make them easily accessible with links.

#. Turn your Point of interest into presentation slides with Presentation Maker.

#. Gain new options that come with the POI, such as hiding your POI from the A+ Course material and making it only appear in the presentation slides.
```

How to use POI?
---------------

1. Point of Interest should work out of the box, but if you ran into problems:

    Try updating your a-plus-rst-tools. If it does not work then try activating the extension by inserting "point_of_interest" in the project conf.py

    (`extensions = ["aplus_setup", "point_of_interest"]`).

> 2. Next step is to start writing reStructuredText content and use POI directives.

Reviewer: Perhaps you could elaborate further and provide a quick example.

Author: Added some relevant links and structured the text better.

> TODO explain here what some of the options do. Like next previous and etc...

Reviewer: I think this orphan `TODO` needs to be done

> In the next chapter

Reviewer: You can remove this.

Point Of Interest (POI)
=======================

.. styled-topic::

  Main questions:
      What are point of interest directives, why and how to use it.

  Topics?

>       What, why and how.

Reviewer: ```suggestion
      Point of interest.
```

  Difficulty:
      Easy

  Laboriousness:
      Around 5 minutes of reading

Introduction to the Point of Interest directive.

What is a point of interest?
----------------------------

Directive for creating a "point of interest" summary block.

> A point of interest is similar to a normal admonition ("coloured info box"), but

Reviewer: How are they similar?. Is it important to mention the similarity?

the point of interest are also linked to each other with next/previous links. The links enable
the user to quickly navigate between different points of interest.

Point of interests may also be used to generate separate lecture slides
(not directly included in the A+ content chapters). This requires a separate
tool called :doc:`Presentation Maker </m10_point_of_interest/03_presentation_maker>`

.. point-of-interest:: This is a point of interest

  And here is the content. More examples in the :doc:`next chapter </m10_point_of_interest/02_examples>`

From now on I will refer to point of interest as a POI.

> Why to use it?
> --------------
> 
> #. Highlight important parts and make them easily accessible with links.
> 
> #. Turn your Point of interest into presentation slides with Presentation Maker.

Reviewer: This is perhaps to little content for a section. Consider making a brief introduction to the `Why to use it` and perhaps making this a subsection.

```suggestion
Why use points of interest?
...........................
Using points of interest in your course may bring some benefits. Firstly, the points of interest can be used to highlight important parts of your course content while linking these elements together in a logical and sequential manner. Finally, the points of interest can be exported as presentation slides with the Presentation Maker tools.
```
**Note:** the suggestion above is just an example, but I think you can write something better.

How to use POI?
---------------

1. Point of Interest should work out of the box, but if you ran into problems:

    - Try updating your a-plus-rst-tools. `Read this, especially the 3rd point. <https://github.com/apluslms/course-templates#installation>`_

>         - Updating should work out with this: :code:`git submodule update`

Reviewer 2: `git submodule update` alone does not update the submodule to the latest available commit. It sets the submodule to the commit that has been set in the main repo for the submodule. However, the a-plus-rst-tools README has instructions about upgrading the rst-tools and you could link to it:
https://github.com/Aalto-LeTech/a-plus-rst-tools/blob/master/README.md#upgrading-the-tools
